docs/developer/guide-for-sps.md

```diff
@@ -4,63 +4,50 @@ sidebar_position: 1
 
-This page contains information about connecting services to
-RCIAM in order to allow user
-login through RCIAM and to receive users' attributes. RCIAM is connected
-to a wide range of academic and social Identity Providers that users can choose
-from in order to access your service.
+This page contains information about connecting services to OpenAIRE Login in
+order to allow user login through OpenAIRE Login and to receive users'
+attributes. OpenAIRE Login is connected to a wide range of academic and social
+Identity Providers that users can choose from in order to access your service.
 
 ## Service Provider integration workflow
 
-To integrate your Service Provider with the RCIAM service, you need to
-create a registration request using the
-[RCIAM Federation Registry Portal](https://federation-demo.rciam.grnet.gr). You
-can also use the Federation Registry portal to request the reconfiguration or
-deregistration of an existing deployed service. Service registration requests
-typically require approval by an administrator. Please refer to the
-[Federation Registry Documentation](https://federation.rciam.grnet.gr/docs) for
-more information.
+To integrate your Service Provider with the OpenAIRE Login service, you need to
+send us an email to `aai` `<AT>` `openaire.eu` indicating your request.
 
 The integration follows a two-step process:
 
-1. Register your Service Provider and test integration with the **demo**
-   instance of RCIAM by selecting the "Demo" integration environment
-   during registration through the RCIAM Federation Registry Portal. The demo
-   instance allows for testing authentication and authorisation through the
-   academic and social Identity Providers connected to RCIAM without
-   affecting the production RCIAM service. Note that while the demo instance
-   has identical functionality to the production instance, no information is
-   shared between the two systems.
-1. Register your Service Provider with the **production** instance of RCIAM
-   by selecting the "Production" integration environment during
-   registration through the RCIAM Federation Registry Portal. The production
-   instance allows access to your service through the academic and social
-   Identity Providers connected to RCIAM. This requires that your service
-   integration has been thoroughly tested during Step 1.
+1. Register your Service Provider and test integration with the **BETA**
+   instance of OpenAIRE Login. The BETA instance allows for testing
+   authentication and authorisation without affecting the production OpenAIRE
+   Login service. Note that while the BETA instance has identical functionality
+   to the production instance, no information is shared between the two systems.
+1. Register your Service Provider with the **production** instance of OpenAIRE
+   Login to allow members of the OpenAIRE User Community to access your service.
+   This requires that the integration has been thoroughly tested during Step 1.
```

Comment:
1. Register your Service Provider with the **production** instance of the
OpenAIRE AAI. The production instance enables access to your service through any of the available  academic and social Identity Providers.
Moving to production requires that your service
integration has been thoroughly tested during Step 1.

Reply:
Done